### DISCUSSION TITLE
opencore may corrupt some information in the bios, such as UEFI

### DISCUSSION THREAD

**Discussion by MokOopsing** [osy86/HaC-Mini#978] [Q&A]
I format the disk I had installed macos and bootcamp windows, and try to install windows, but there seems to be some issues, now my nuc cannot detect winpe and macos install usb stick. when I reset bios(with secure boot checked), it can detect win10 install usb stick, but when I install win10 and pull off usb stick, it still cannot boot to win10, there always show macos x and opencore in boot pick menu(F10) even no macos partion or usb.

I  found others got same issue before, but no solution posted https://www.zhihu.com/question/477452431

![F6AAAE17-0B23-4EE3-97C6-934E5F7BE456](https://user-images.githubusercontent.com/39441028/153779971-8c7eb714-483b-410d-b59d-f07d0e10c359.jpeg)

**Comment by MokOopsing** [comment]
@goodbest @osy Could you please provide me any idea? much thx

**Reply by goodbest** [reply]
If you have two ssd, I' d suggest you disable one of them in BIOS during the installation to make it easier to debug.

If you only have one ssd, make sure
- Reset your bios to remove the previous opencore boot entry (I see you have done this already, just to verify it again)
- Completed format your ssd including the invisible EFI partition. (Instead of clicking "next" all along, you may have to manually remove partition, format disk, and new partition during windows instalation)
- Use F12 to entry one-time boot menu instead of the default boot menu
- reset NVRAM

If the above won't work, you can try install Ubuntu, winPE, or another MacOS first.

**Reply by MokOopsing** [reply]
> If you have two ssd, I' d suggest you disable one of them in BIOS during the installation to make it easier to debug.
> 
> If you only have one ssd, make sure
> 
> * Reset your bios to remove the previous opencore boot entry (I see you have done this already, just to verify it again)
> * Completed format your ssd including the invisible EFI partition. (Instead of clicking "next" all along, you may have to manually remove partition, format disk, and new partition during windows instalation)
> * Use F12 to entry one-time boot menu instead of the default boot menu
> * reset NVRAM
> 
> If the above won't work, you can try install Ubuntu, winPE, or another MacOS first.

I disabled the 2nd ssd, and I made it boot to win10 install, the I deleted all partions on the ssd and create new partion, then I installed win10 and it reboot, and failed to complete the installation, still booted to usb, and boot picker menu still showed mac os x and opencore, these are what I did before

Is F12 a typo？ do you mean F10?

And how could I reset nvram if I cannot boot to opencore this time

**Reply by MokOopsing** [reply]
WINPE cannot be detect either

**Comment by MokOopsing** [comment]
After I did the bios setting for macos, there is always "mac os x" showing in boot picker menu, even no macos efi and usb stick, and I  had already fully format my disk. so I have to apply for after-sales service, cause I tried reset bios by removing security jamper and bios battery, none of them could clear mac os x / opencore boot option

**Reply by MokOopsing** [reply]
And when I plugged a macos install usb stick with patch by hac-mini, it could never detect it

So I think opencore may write sth into bios that hard to clear , or it may damage the board

**Comment by MokOopsing** [comment]
My Hades Canyon was eventually returned to the factory for repair, but it was not repaired, and I finally got a refund, although only eighty percent of the original price.

Originally I was worried that you would end support before it was retired, but I did not expect it to be retired early.

Finally, thank you very much for your dedication to this project, and I bid you all farewell.

@goodbest @osy

**Reply by lzxjimmy** [reply]
兄弟，两天前我也遇到这个问题。不过你试下，长按Alt可以呼出oc菜单，但是还是看不到引导安装盘。后来我用一个usb2.0的u盘做恢复安装，机器直接可以认到。而usb3.0的u盘，制作方法一样，机器就是死活认不到。原因不明。

**Reply by MokOopsing** [reply]
> 兄弟，两天前我也遇到这个问题。不过你试下，长按Alt可以呼出oc菜单，但是还是看不到引导安装盘。后来我用一个usb2.0的u盘做恢复安装，机器直接可以认到。而usb3.0的u盘，制作方法一样，机器就是死活认不到。原因不明。

已经拿钱了，机器不在手上了，当时是可以用u盘装上win10,但是启动不了，磁盘此前已经只留一个且被完全格式化过了，毕竟是京东愿意退钱都修不好了

**Comment by xiaooloong** [comment]
你在哪里买的呀，现在返厂变成退款了？

**Reply by MokOopsing** [reply]
某东，因为intel也没货了
